src/pages/ProjectMembers.jsx

```diff
@@ -1,9 +1,56 @@
-import React from 'react'
+import React, { useEffect, useState } from "react";
+import { getProjectMembers } from "../services/teams";
 
 const ProjectMembers = () => {
+  const [teamMembers, setTeamMembers] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    getTeamMembers();
+  }, []);
+
+  const getTeamMembers = async () => {
+    try {
+      setLoading(true)
+      const members = await getProjectMembers();
+      setTeamMembers(members);
+      setLoading(false)
+    } catch (error) {
+      setLoading(false)
+      console.log(error);
+    }
+  };
+
   return (
-    <div>ProjectMembers</div>
-  )
-}
+    // team members page
+    <div className="flex items-center bg-white">
+      <div>
+        <p className="font-bold text-blue-900 text-4xl mt-8 ">
+          Meet the Team...
+        </p>
+        <div className="flex flex-wrap mt-8 mx-16 font-bold">
+          {teamMembers.map((member) => (
+            <div
```

Comment:
can you extract the Team Card as a component and and put it in the components folder. Then import it here and passing the member data as props to it . 

`<TeamCard team={team} /> `